### DISCUSSION TITLE
Need also Child arrow

### DISCUSSION THREAD

**Discussion by silicahd** [givanz/Vvveb#66] [Ideas]
Please see here we also need the child arrow. 

If the code is like this <a href="xxxx"><img url="" ></a>  you can not get to the image settings. 

![image](https://github.com/givanz/Vvveb/assets/7761519/e6c4aa1c-b0e9-48c4-b8cf-1dded6d54d04)

Every DIV should allow to have settings like display, padding, margins etc. It seems to only work if it is container. In this example there is no way to move the image to the center. 

![image](https://github.com/givanz/Vvveb/assets/7761519/7e679b18-fa72-48f8-a109-e76b88a07328)

**Comment by givanz** [comment]
Grid and row components were not extending base component and didn't include style options.

I changed them to extend base to inherit style in the last update https://github.com/givanz/Vvveb/commit/040d1ad3b9dbe87558c55593d854b850dd250a58, changed files in `public/js`.

While an element can have one parent it can have many children and a button to select child elements can't be added in the same way as parent button.

You can select the inner most element in a section and then use the breadcrumb to select a parent/grand parent element.

Probably a hierarchy navigator window is a better option to help better select elements.
